strawberryfields/decompositions.py

```diff
@@ -202,6 +212,47 @@ def clements(V, tol=1e-11):
 
     return tilist, tlist, np.diag(localV)
 
+def clements_eq5(V, tol=11):
```

Comment:
Is there an alternative function name we could use that is a bit more descriptive here? Maybe, `clements_phase_end` or `clements_final_phase`. Those are pretty bad, so feel free to use your own suggestions!